### PR TITLE
ui/admin: add service metrics data table

### PR DESCRIPTION
**Pull request by KatieMSB**
- [x] Identified the issue which this PR solves.
- [x] Read the [**CONTRIBUTING**](https://github.com/target/goalert/blob/master/CONTRIBUTING.md) document.
- [x] Code builds clean without any errors or warnings.
- [x] Added appropriate tests for any new functionality.
- [x] All new and existing tests passed.
- [x] Added comments in the code, where necessary.
- [x] Ran `make check` to catch common errors. Fixed any that came up.

**Description:**
This PR adds part of the service metrics ui work that includes overall metrics for all services and a table of all services.
Metrics include:
- Total Services
- Services with empty Escalation Policies
- Services without Integration Keys and Heartbeat Monitors
- Services with stale Alerts (acknowledged alerts older than 2 months)
- Services nearing or at the Unacknowledged Alert Limit

**Which issue(s) this PR fixes:**
Part of #3057

**Out of Scope:**
- filtering
- integration key/escalation policy target metrics breakdowns

**Screenshots:**

Screenshot 2023-10-25 at 10 20 07 AM

### CODE REVIEW

File: web/src/app/admin/admin-service-metrics/AdminServiceTable.tsx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+      valueGetter: (params: GridValueGetterParams) => {
+        return params.row.escalationPolicy?.name || ''
+      },
+      renderCell: (params: GridRenderCellParams<GridValidRowModel>) => {
+        if (
+          params.row.escalationPolicy.id &&
+          params.row.escalationPolicy.name
+        ) {
+          return (
+            <AppLink to={`/services/${params.row.escalationPolicy.id}`}>
```
Comment (tony-tvu) [inline web/src/app/admin/admin-service-metrics/AdminServiceTable.tsx:146]:
```suggestion
            <AppLink to={`/escalation-policies/${params.row.escalationPolicy.id}`}>
```

**Review by tony-tvu — CHANGES_REQUESTED** [review]

File: web/src/app/admin/admin-service-metrics/AdminServiceTable.tsx (hunk 1 above)
Comment (KatieMSB) [reply web/src/app/admin/admin-service-metrics/AdminServiceTable.tsx:146]: good catch 👍🏽

**Review by tony-tvu — APPROVED** [review]
LGTM

**Review by tony-tvu — DISMISSED** [review]

**Review by Forfold — CHANGES_REQUESTED** [review]
As a subheader under Services, could we maybe have some loading text, such as "Loading... This may take a minute."

Also, this could maybe be iterated on later, but thinking about the top cards to be more responsive for various browser sizes. Maybe switch to `xs={6} sm={2.4}` or something?

Last thing, is perhaps setting the height to the top cards to 100% of their container, so their heights all match:

Screenshot 2023-10-26 at 10 48 18 AM

**Review by Forfold — APPROVED** [review]

**Review by ethan-haynes — APPROVED** [review]
LGTM 👍

**Pull request merged**